src/user/UserService.ts

```diff
@@ -1,15 +1,23 @@
 import bcrypt from 'bcrypt';
 import User from './User';
+import crypto from 'crypto';
+
+const generateToken = (length: number): string => {
```

Comment:
This is a function to generate random toke with the use of crypto library built-in with node